Project Requirements Document.md

```diff
@@ -0,0 +1,27 @@
+﻿
+|    **PROJECT REQUIREMENTS DOCUMENT**   |                                                                                                                                                                                                              |
+|:--------------------------------------:| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|            **Project Name**            | Threewide                                                                                                                                                                                                    |
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our primary goal is to teach T spin by asking the user to perform n amount of them in small gmaes embedded on Threewide. It also includes teaching of Super Rotation System Kick Table, DAS and ARP etc.                                                                                                                                          |
```

Comment:
Nitpick: Games is misspelled here

Reply:
Also we won't be teaching SRS DAS and ARR. This is just what is included in the modern Tetris ruleset

Reply:
Fixed